app/views/posts/_expanded.html.erb

```diff
@@ -460,5 +462,3 @@
 <% if moderator? || current_user&.has_post_privilege?('flag_curate', post) %>
   <%= render 'posts/post_tools', post: post %>
 <% end %>
-
-<%= render 'comments/new_thread_modal', post: post %>
```

Comment:
Where is the new thread modal being added into the page, if not here? I can't seem to see anywhere else.

Reply:
Check in latest expanded.html.erb commit. The new thread modal is on 453 line.
```
<% else %>
                <% if post.locked? %>
                  <p class="has-color-red-500"><i class="fa fa-lock"></i> Comments are disabled on locked posts, but as a moderator you are exempt from that block.</p>
                <% elsif post.comments_disabled %>
                  <p class="has-color-red-500"><i class="fa fa-lock"></i> Comments have been disabled on this post, but as a moderator you are exempt from that block.</p>
                <% end %>
                <a class="button is-muted is-small js-new-thread-link new_thread" data-post="<%= post.id %>">Start new comment thread</a>
              <% end %>
                <%= render 'comments/new_thread_modal', post: post %>
            <% end %>
```